cli/index.ts

```diff
@@ -206,8 +171,53 @@ const timeoutOption: INamedOption<yargs.Options> = {
   }
 };
 
-const defaultDatabaseOptionName = "default-database";
-const defaultLocationOptionName = "default-location";
+const defaultDatabaseOption: INamedOption<yargs.Options> = {
+  name: "default-database",
+  option: {
+    describe: "The default database to use. For BigQuery, this is a Google Cloud Project ID.",
```

Comment:
we should clarify (for all of these) that they override any settings in `dataform.json`

Reply:
Thread above - this is for init, so does not override anything, and is a required flag in this case.